bower.json

```diff
@@ -17,7 +18,9 @@
     "moment": "~2.10.6",
     "junction": "theleagueof/junction",
     "publiclab-editor": "~0.1.0",
-    "leaflet-d3": "^0.3.8"
+    "leaflet-d3": "^0.3.8",
+    "woofmark": "~4.2.0",
+    "horsey": "3.0.0"
```

Comment:
great catch, thanks!

On Tue, Aug 16, 2016 at 5:38 PM, Ananya Maiti <notifications@github.com>
wrote:

> In bower.json
> <https://github.com/publiclab/plots2/pull/664#discussion_r75025719>:
>
> > @@ -17,7 +18,9 @@
> >      "moment": "~2.10.6",
> >      "junction": "theleagueof/junction",
> >      "publiclab-editor": "~0.1.0",
> > -    "leaflet-d3": "^0.3.8"
> > +    "leaflet-d3": "^0.3.8",
> > +    "woofmark": "~4.2.0",
> > +    "horsey": "3.0.0"
>
> Yes the rich editor does work fine but some of the css was broken when i
> was using it in my local machine. I figured out it was due to the missing
> css files in the packages. ᐧ On Wed, Aug 17, 2016 at 2:50 AM, Jeffrey
> Warren <notifications@github.com> wrote:
> … <#m_-634136899203549829_>
> In bower.json <#664 (comment)
> <https://github.com/publiclab/plots2/pull/664#discussion_r75022641>>: >
> @@ -17,7 +18,9 @@ > "moment": "~2.10.6", > "junction":
> "theleagueof/junction", > "publiclab-editor": "~0.1.0", > - "leaflet-d3":
> "^0.3.8" > + "leaflet-d3": "^0.3.8", > + "woofmark": "~4.2.0", > +
> "horsey": "3.0.0" Haha, ok, excellent-- but what was not working? The rich
> editor works fine, doesn't it? … <#m_7002025029968671426_> On Aug 16, 2016
> 5:15 PM, "Ananya Maiti" ***@***.***> wrote: In bower.json <#664
> <https://github.com/publiclab/plots2/pull/664> (comment) <#664 (comment)
> <https://github.com/publiclab/plots2/pull/664#discussion_r75021691>>>: >
> @@ -17,7 +18,9 @@ > "moment": "~2.10.6", > "junction":
> "theleagueof/junction", > "publiclab-editor": "~0.1.0", > - "leaflet-d3":
> "^0.3.8" > + "leaflet-d3": "^0.3.8", > + "woofmark": "~4.2.0", > +
> "horsey": "3.0.0" Yes the javascript modules are indeed complied into
> PublicLab.editor.js but the css files aren't so we need these modules. The
> rich editor template links the stylesheets of those packages without which
> the css gets broken. ᐧ … <#m_8393582544900688499_> On Wed, Aug 17, 2016 at
> 2:29 AM, Jeffrey Warren ***@***.***> wrote: In bower.json <#664
> <https://github.com/publiclab/plots2/pull/664> <#664
> <https://github.com/publiclab/plots2/pull/664>> (comment) <#664
> <https://github.com/publiclab/plots2/pull/664> (comment) <#664 (comment)
> <https://github.com/publiclab/plots2/pull/664#discussion_r75018921>>>>: >
> @@ -17,7 +18,9 @@ > "moment": "~2.10.6", > "junction":
> "theleagueof/junction", > "publiclab-editor": "~0.1.0", > - "leaflet-d3":
> "^0.3.8" > + "leaflet-d3": "^0.3.8", > + "woofmark": "~4.2.0", > +
> "horsey": "3.0.0" These are actually compiled into publiclab-editor using
> browserify, so they're not needed in this bower.json file. — You are
> receiving this because you authored the thread. Reply to this email
> directly, view it on GitHub <https://github.com/publiclab/
> plots2/pull/664/files/31d538c99234a755f3d9868bd56022d400c0e371#r75018921>,
> or mute the thread <https://github.com/notifications/unsubscribe-auth/
> AKACR9950kpTkYRKWqAeVf3h6nfeqVhJks5qgiSfgaJpZM4JlHKF> . — You are
> receiving this because you were mentioned. Reply to this email directly,
> view it on GitHub <https://github.com/publiclab/plots2/pull/664/files/
> 31d538c
> <https://github.com/publiclab/plots2/commit/31d538c99234a755f3d9868bd56022d400c0e371>#r75021691>,
> or mute the thread < https://github.com/notifications/unsubscribe-
> auth/AABfJ4pAz9- MJnwi16cfL18yINn_uUQwks5qgih1gaJpZM4JlHKF> . — You are
> receiving this because you authored the thread. Reply to this email
> directly, view it on GitHub <https://github.com/publiclab/
> plots2/pull/664/files/31d538c99234a755f3d9868bd56022d400c0e371#r75022641>,
> or mute the thread <https://github.com/notifications/unsubscribe-
> auth/AKACRx2XdOqSx_Z8hDikjKty7OvGqtdoks5qgimygaJpZM4JlHKF> .
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/664/files/31d538c99234a755f3d9868bd56022d400c0e371#r75025719>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJ8EQQN1Kze-wf3I5Vp0wM6aiCGxLks5qgi3ugaJpZM4JlHKF>
> .
>